packages/embark-ui/src/components/Transactions.js

```diff
@@ -45,6 +45,15 @@ const Transactions = ({transactions, contracts, changePage, currentPage, numberO
                   <strong>Type</strong>
                   <div>{transaction.to ? "Contract Call" : "Contract Creation"}</div>
                 </Col>
+                <Col md={6}>
+                  <strong>Mined on:</strong>
+                  <div>
+                    {isToday(new Date(transaction.timestamp * 1000)) ?
```

Comment:
The functions need to be imported, I think. I’m fairly sure that’s why Travis is failing. I’m commenting from mobile so can’t test atm.

Reply:
I could swear I imported those, otherwise the whole feature wouldn't have worked. Have zero idea how I ended up not committing the imports...

Will update the PR. Thanks for checking M!